lib/rules/one-var.js

```diff
@@ -53,6 +84,34 @@ module.exports = function(context) {
         }
     }
 
+    /**
+     * Determines if there is more than one let statement in the current scope.
+     * @returns {boolean} Returns true if it is the first let declaration, false if not.
```

Comment:
I'm trying to follow this, but since each block begins with `let` and `const` both `false` (in `startBlock`), doesn't this return `false` the first time and `true` every time thereafter?